src/jupyterhub_sphinx_theme/__init__.py

```diff
@@ -29,7 +29,7 @@ def set_config_defaults(app):
     # Default jupyter favicon
     if not _config_provided_by_user(app, "html_favicon"):
         config.html_favicon = (
-            "https://github.com/jupyterhub/jupyterhub-sphinx-theme/raw/master/src/jupyterhub_sphinx_theme/theme/jupyterhub-sphinx-theme/static/favicon.png"
+            "https://github.com/jupyterhub/jupyterhub-sphinx-theme/raw/main/src/jupyterhub_sphinx_theme/theme/jupyterhub-sphinx-theme/static/favicon.ico"
```

Comment:
@choldgraf Is there a simple way to have the theme automatically copy the default icon to under `static` in the deployed RTD site, instead of having a cross-site link? This will reduce the likelihood of it being blocked by users with strict browser settings.

Reply:
You could bundle the icon with the theme itself and link it locally. Or you could make the theme download the icon at docs build time and cache it or something.

Reply:
To bundle the icon somehow instead of linking out to github is tracked by #3.

I opted to not go for that in this PR as I didn't knew how to do this without further research. Can we go for a merge of this PR as it is as an incremental improvement?